src/context/WalletProvider/KeepKey/hooks/useKeepKeyVersions.ts

```diff
@@ -30,26 +35,30 @@ interface VersionStatus {
 interface Versions {
   bootloader: VersionStatus
   firmware: VersionStatus
+  updaterUrl: string
```

Comment:
I was struggling a bit on this, but I ended thinking that the `updaterUrl` is hardly linked to versioning, probably nitpick but if you feel like it should be added somewhere else, just tell me!

Reply:
If we want to decouple it from the `Version` type, we could get `useKeepKeyVersions` to either return a tuple or an object:
`const [versions, updaterUrl] = useKeepKeyVersions()` or
`const {versions, updaterUrl} = useKeepKeyVersions()`

Reply:
@gomesalexandre I did it, seems better